### PR TITLE
add(pattern): shell 1

### PR DESCRIPTION
**Pull request by deepakgithub11**
This PR adds a new shell 1 pattern for WPUI

### CODE REVIEW

**Comment by deepakgithub11** [comment]
Ready For Review

File: src/categories/index.js [hunk 1]
```diff
@@ -7,5 +7,6 @@ export { default as GridList } from './grid-list';
 export { default as Lists } from './lists';
 export { default as Navigations } from './navigations';
 export { default as PageHeading } from './page-heading';
+export { default as Shell } from './shell';
```
Comment (ajitbohra) [inline src/categories/index.js:10]:
```suggestion
export { default as Shells } from './shells';
```

File: src/categories/shell.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+ * Internal dependencies.
+ */
+import * as patterns from '../patterns';
+
+
+/**
+ * Render Shells
+ */
+
+function Shell() {
```
Comment (ajitbohra) [inline src/categories/shell.js:16]:
```suggestion
function Shells() {
```

File: src/categories/shell.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  );
+};
+
+Shell.meta = {
+  title: 'Shell',
+  path: '/shell',
+  patterns: patterns.Shell,
+};
+
+export default Shell;
```
Comment (ajitbohra) [inline src/categories/shell.js:41]:
```suggestion
export default Shells;
```

File: src/categories/shell.js
Comment (ajitbohra) [inline src/categories/shell.js:1]: fille name should be shells.js

File: src/patterns/Shell/shell1.js
Comment (ajitbohra) [inline src/patterns/Shell/shell1.js:1]: folder should be Shells

File: src/patterns/index.js [hunk 4]
```diff
@@ -8,6 +8,7 @@ export * as EmptyState  from './EmptyState';
 export * as ActionPanels  from './ActionPanels';
 export * as Feeds  from './Feed';
 export * as Forms  from './Form';
+export * as Shell  from './Shell';
```
Comment (ajitbohra) [inline src/patterns/index.js:11]:
```suggestion
export * as Shell  from './Shells';
```

**Review by ajitbohra — CHANGES_REQUESTED** [review]

File: src/categories/shell.js [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+      <rect x="42.4808" y="46.8375" width="24.4252" height="4.44094" rx="2.10236" fill="#D6E4FF" />
+      <rect x="79.1185" y="46.8375" width="23.315" height="4.44094" rx="2.10236" fill="#D6E4FF" />
+      <rect x="114.646" y="46.8375" width="24.4252" height="4.44094" rx="2.10236" fill="#D6E4FF" />
+      <rect x="151.284" y="46.8375" width="24.4252" height="4.44094" rx="2.10236" fill="#D6E4FF" />
+    </svg>
+  );
+};
+
+Shell.meta = {
```
Comment (ajitbohra) [inline src/categories/shell.js:35]:
```suggestion
Shells.meta = {
```

File: src/categories/shell.js [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+  title: 'Shell',
+  path: '/shell',
+  patterns: patterns.Shell,
```
Comment (ajitbohra) [inline src/categories/shell.js:38]:
```suggestion
  patterns: patterns.Shells,
```

File: src/categories/shell.js [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+      <rect x="79.1185" y="46.8375" width="23.315" height="4.44094" rx="2.10236" fill="#D6E4FF" />
+      <rect x="114.646" y="46.8375" width="24.4252" height="4.44094" rx="2.10236" fill="#D6E4FF" />
+      <rect x="151.284" y="46.8375" width="24.4252" height="4.44094" rx="2.10236" fill="#D6E4FF" />
+    </svg>
+  );
+};
+
+Shell.meta = {
+  title: 'Shell',
+  path: '/shell',
```
Comment (ajitbohra) [inline src/categories/shell.js:37]:
```suggestion
  path: '/shells',
```

File: src/categories/shell.js (hunk 3 above)
Comment (ajitbohra) [inline src/categories/shell.js:41]:
```suggestion
export default Shells
```

File: src/patterns/Shells/shell1.js [hunk 8]
```diff
@@ -0,0 +1,98 @@
...
+        { icon: plugins, label: 'Addons' },
+        { icon: key, label: 'License' },
+        { icon: external, label: 'Support' },
+        { icon: trendingUp, label: 'Go Pro', variant: 'primary' }
+    ];
+    return (
+        <Card>
+            <CardHeader>
+                <HStack>
+                    <HStack expanded={false} style={{ marginLeft: '5px' }}>
```
Comment (Verma-Punit) [inline src/patterns/Shells/shell1.js:43]: Remove the HStack do not wrap the img tag within it

File: src/patterns/Shells/shell1.js [hunk 9]
```diff
@@ -0,0 +1,98 @@
...
+                        <img width='100px' style={{minWidth:'100px'}} src="https://raw.githubusercontent.com/lubusIN/wpui/main/src/img/logo.png"></img>
```
Comment (Verma-Punit) [inline src/patterns/Shells/shell1.js:44]: Remove the HStack do not wrap the img tag within it

File: src/patterns/Shells/shell1.js [hunk 10]
```diff
@@ -0,0 +1,98 @@
...
+                                    variant={btn.variant}
+                                    style={btn.label === 'Go Pro' ? { display: 'flex', justifyContent: 'center' } : {}}
+                                >
+                                    {btn.label}
+                                </Button>
+                            ))}
+                        </VStack>
+                )}
+                <CardBody style={{ backgroundColor: '#F0F0F1', width: '100vw' }}>
+                    <Heading style={{ padding: '20px', marginLeft: '10px' }}>Dashboard</Heading>
```
Comment (Verma-Punit) [inline src/patterns/Shells/shell1.js:79]: Remove the inline style it will affect responsiveness.

File: src/patterns/Shells/shell1.js [hunk 11]
```diff
@@ -0,0 +1,98 @@
...
+                    <Card style={{ backgroundColor: '#F0F0F1', height: '300px', margin: '30px 30px', borderRadius: '10px' }}>
```
Comment (Verma-Punit) [inline src/patterns/Shells/shell1.js:80]: Remove the margin style it will affect responsiveness.

**Pull request merged**